bin/morgue.ts

```diff
@@ -7380,7 +7383,7 @@ function ruleStatus(rule, argv, spaces) {
   const exp_data = rule.criteria.reduce((exp_data, crit) => {
     if (exp_data.off === 0) {
       exp_data.off = crit.value;
-    } else if (exp_off > crit.value) {
```

Comment:
Bug fix: `exp_off` is not declared anywhere, but there is `exp_data` with a `off` field, so `exp_data.off` is almost certainly what was intended.